gym_sts/test_valid_actions.py

```diff
@@ -61,7 +61,7 @@ def main():
             raise ValueError("No %svalid actions!" % ("" if want_valid else "in"))
 
         action = rng.choice(actions)
-        print(action)
+        print(repr(action))
```

Comment:
I find it strange that we need to call `repr` here instead of just `str`. Do you know what's going on there?

Reply:
It's necessary because of a design decision by pydantic: https://github.com/pydantic/pydantic/pull/884#issue-505232916

I agree it seems a little odd, but I guess their view is that the class name isn't meant to be presented to humans. The purpose of pydantic is to parse, validate, and represent a data structure. We have pydantic models that have no fields, which is probably unusual.